index.bs

```diff
@@ -2614,6 +2616,18 @@ and an optional [=aggregatable report=] <dfn for="generate null reports and assi
     1. [=list/Append=] |report| to |reports|.
 1. Run [=assign private state tokens=] with |reports| and |trigger|.
 
+<h3 id="deferring-trigger-attribution">Deferring trigger attribution</h3>
+
+To <dfn>maybe defer trigger attribution</dfn> given an [=attribution trigger=] |trigger|:
+
+1. Let |navigation| be the navigation that landed on the document from which |trigger|'s registration was initiated.
+1. If |navigation| is empty, Return.
+1. Let |sources| be all source registrations initiated alongside |navigation|.
```

Comment:
Maybe clarify that the source registrations were ones originating from background attributionsrc requests that were initiated by `|navigation|`. IIUC foreground ones don't need to be handled specially here.